lm_eval/tasks/multiple.py

```diff
@@ -80,12 +80,16 @@ class GeneralMultiPLE(Task):
 
     DATASET_PATH = "nuprl/MultiPL-E"
     DATASET_NAME = None
+    DATASET_REVISION = "5d2abbb8ced9a0e37db985c47d24c24f45a16655"
```

Comment:
This is the current latest main: https://huggingface.co/datasets/nuprl/MultiPL-E/tree/main